CONTRIBUTING.md

```diff
@@ -10,6 +10,17 @@ OPI uses GitHub to manage reviews of pull requests.
 
 * Be sure to sign off on the [DCO](https://github.com/probot/dco#how-it-works).
 
+## Architecture Changes
+
+Changes to project wide goals or scope, subgroup goals or scope, or major
+architectural changes shall be:
+
+* Submitted via PR to the appropriate repo
+* Labeled with the `architecture` label
+* Broadcast to the `opi-dev` mailing list
+* Left open for review for at least 2 days
```

Comment:
We have somewhat non-overlapping working days and time zones (for instance, an email to opi-dev about this pull request came into my inbox after 9pm on my last workday of the week, and I only noticed it after a long work day the following week).

I understand the desire to be able to move faster, but let's make this time window somewhat longer for architecture changes. These are material changes, they will not be made often, and they can be left to bake slightly longer. I propose making this 4 calendar days.

Reply:
4 days sounds reasonable

Reply:
I'm also ok with 4 days.

Reply:
4 days is fine - should we clarify if this is workdays vs calendar days?

Reply:
I agree Mark - work days. If that is the consensus, the doc should be updated before the vote on Thursday.